modules/custom-status/custom-status.php

```diff
@@ -1830,11 +1840,14 @@ function column_name( $item ) {
 		$actions = array();
 		$actions['edit'] = "<a href='$item_edit_link'>" . __( 'Edit', 'edit-flow' ) . "</a>";
 		$actions['inline hide-if-no-js'] = '<a href="#" class="editinline">' . __( 'Quick&nbsp;Edit' ) . '</a>';
-		if ( $item->slug != $this->default_status )
-			$actions['make_default'] = sprintf( '<a href="%1$s">' . __( 'Make&nbsp;Default', 'edit-flow' ) . '</a>', $edit_flow->custom_status->get_link( array( 'action' => 'make-default', 'term-id' => $item->term_id ) ) );
-
-		if ( $item->slug != $this->default_status )
-			$actions['delete delete-status'] = sprintf( '<a href="%1$s">' . __( 'Delete', 'edit-flow' ) . '</a>', $edit_flow->custom_status->get_link( array( 'action' => 'delete-status', 'term-id' => $item->term_id ) ) );
+		$actions['make_default'] = sprintf( '<a href="%1$s">' . __( 'Make&nbsp;Default', 'edit-flow' ) . '</a>', $edit_flow->custom_status->get_link( array( 'action' => 'make-default', 'term-id' => $item->term_id ) ) );
+		
+		// Disable delete/make-default actions for draft status
```

Comment:
There seems to be an if inside an if but we only do something if they are both true, could we combine them?
I'm also not 100% clear on what this is doing. How are we preventing the make default action?
Why are we preventing draft from being a default status?

Reply:
Bad PHP comment. Initially I disabled making default and came to a conclusion that it shouldn't be prevented, forgot to edit the comment.

Reply:
The comment is better but we still have nested ifs, not a big deal but it's just strange to read :)

Reply:
Whoa. After moving all the code around I didn't notice an unnecessary nested if statement. Fixed.